### PR TITLE
Pyserini repro doc for MS MARO v2

### PR DESCRIPTION
**Pull request by crystina-z**
- bm25 baselines
- scripts to generate doc and passage training triplet id
~- pointer to capreolus about monobert / maxp repro doc?~
~- pointer to dense / sparse retrieval?~

### CODE REVIEW

File: docs/experiments-msmarco-v2.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Index size can be reduced by removing the options `-storePositions`, `-storeDocvectors`, `-storeRaw` as appropriate.
+For reference:
+
++ Without any of the three above option, index size reduces to 9.4 GB.
++ With just `-storeRaw`, index size reduces to 73 GB. This setting contains the raw JSON document, which makes it suitable for use as first-stage retrieval to support downstream rerankers. Bloat compared to compressed size of raw collection is due to support for per-document random access; evidently, the JSON docs don't compress well.
+
+Perform a run on the dev queries:
+
+```
+python -m pyserini.search -index indexes/msmarco-doc-v2 \
```
Comment (MXueguang) [inline docs/experiments-msmarco-v2.md:80]: same as above

File: docs/experiments-msmarco-v2.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+It's big because it includes postions (for phrase queries), document vectors (for relevance feedback), and a complete copy of the collection itself.
+The index size can be reduced by removing the options `-storePositions`, `-storeDocvectors`, `-storeRaw` as appropriate.
+For reference:
+
++ Without any of the three above option, index size reduces to 12 GB.
++ With just `-storeRaw`, index size reduces to 47 GB. This setting contains the raw JSON document, which makes it suitable for use as first-stage retrieval to support downstream rerankers. Bloat compared to compressed size of raw collection is due to support for per-document random access.
+
+
+```
+python -m pyserini.search -index indexes/msmarco-passage-v2 \
```
Comment (MXueguang) [inline docs/experiments-msmarco-v2.md:42]: pyserini need double dash, and we dont need topicreader here.
e.g.
```
MSMARCO_V2=/home/xueguang/scratch2
python -m pyserini.search --index ${MSMARCO_V2}/index_msmarco_v2_passage \
 --topics ${MSMARCO_V2}/dev/passv2_dev_queries.tsv \
 --output ${MSMARCO_V2}/dev/run.msmarco-pass-v2.dev.txt \
 --bm25 --hits 100 --batch-size 36 --threads 12
```

**Review by MXueguang — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-v2.md (hunk 2 above)
Comment (MXueguang) [reply docs/experiments-msmarco-v2.md:42]: `batch-size` and `threads` are optional

File: docs/experiments-msmarco-v2.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+ -topicreader TsvInt -topics collections/passv2_dev_queries.tsv \
+ -output runs/run.msmarco-pass-v2.dev.txt \
+ -bm25 -hits 100
```
Comment (ronakice) [inline docs/experiments-msmarco-v2.md:45]: Can we do hits = 1K for passage and 100 for document?

**Review by ronakice — CHANGES_REQUESTED** [review]
One comment

File: docs/experiments-msmarco-v2.md (hunk 3 above)
Comment (ronakice) [reply docs/experiments-msmarco-v2.md:45]: @MXueguang what do you think?

**Review by MXueguang — APPROVED** [review]
lgtm

**Review by ronakice — APPROVED** [review]
lgtm

**Pull request merged**